virtcontainers/container.go

```diff
@@ -432,6 +432,19 @@ func (c *Container) rollbackFailingContainerCreation() {
 	}
 }
 
+func (c *Container) checkBlockDeviceSupport() bool {
```

Comment:
This is part of #138, why include it here?

Reply:
@amshinde ^^  ?

Reply:
@bergwolf Without this factorization we need to introduce changes in the noopAgent implementation to support capabilities related to block device support since this code was moved into hotplugDrive which we are testing here. I think this is not necessary since we anyways need to factorize and reuse in #138. So I have moved it here to avoid the unnecessary implementation changes for noopAgent.